api/query/types_test.go

```diff
@@ -0,0 +1,69 @@
+// +build small medium
```

Comment:
This file is not really a test. And the presence of two tags is more like a hack to include this file when building both small and medium tests.

A more common paradigm seen in Go standard library for this kind of testing utilities is to create a new subpackage, e.g. `api/query/querytest` and to import the package in the test cases.